docs/src/quickstart/index.md

```diff
@@ -1,8 +1,24 @@
-## Quickstart
+# Quickstart
 
-Let's get started with Fe! In this chapter you will learn how to install the Fe compiler and write your first contract.
+Let's get started with Fe! 
+
+In this section you will learn how to write and deploy your first contract.
 
-* [Installation](installation.md)
 * [Writing your first contract](first_contract.md)
 * [Deploying a contract to a testnet](deploy_contract.md)
-* [Using projects](projects.md)
+
+## Download and install Fe
+
+Before you dive in, you need to download and install Fe. 
```

Comment:
Not necessary to update this PR but I just wanted to make you aware that we recently (basically yesterday) automated our homebrew installation mechanics. Basically if you have homebrew installed (Mac or Windows) you can run `brew install fe-lang/tap/fe` now to install fe and expose it as `fe` command without any further hassle.

We don't have to maintain these manually so they should automatically keep in sync as we cut new releases.

Reply:
oh that's awesome. I wasn;t aware - ok I will add this information to this page and the installation guide in the next pr.